scss/mixins/_buttons.scss

```diff
@@ -56,8 +56,6 @@
 
 @mixin button-outline-variant($color, $color-hover: color-yiq($color), $active-background: $color, $active-border: $color) {
   color: $color;
-  background-color: transparent;
-  background-image: none;
```

Comment:
I believe the `background-image: none` was for overriding some browser defaults on the button. I can't recall which browser or version, but I recall having to set this in order for custom backgrounds to work properly. Looking around, seems fine now.

Reply:
Ah, yes: https://github.com/twbs/bootstrap/blob/2c2ac3356425e192f7537227508c809a14aa5850/less/buttons.less#L17

Reply:
Yeah, but do we support such old versions anymore? I too remember this issue but it was very old.

Reply:
The `.btn` elements don't have a background-image either, that's why I assumed it was save to remove this.